### PR TITLE
[Site] Clean LiveMemory styles

### PR DESCRIPTION
**Pull request by smnandre**
Clean CSS files

### CODE REVIEW

File: ux.symfony.com/src/LiveMemory/Asset/CssCompiler.php [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+class CssCompiler implements AssetCompilerInterface
+{
+    public function supports(MappedAsset $asset): bool
+    {
+        return 'css' === $asset->publicExtension && str_starts_with($asset->logicalPath, 'styles/demos/live-memory');
+    }
+
+    public function compile(string $content, MappedAsset $asset, AssetMapperInterface $assetMapper): string
+    {
+        $foo = preg_replace_callback('#@import (?:"|\')(.*)(?:\'|");#m', function ($matches) use ($asset) {
```
Comment (Kocal) [inline ux.symfony.com/src/LiveMemory/Asset/CssCompiler.php]:
```suggestion
        return preg_replace_callback('#@import (?:"|\')(.*)(?:\'|");#m', function ($matches) use ($asset) {
```

File: ux.symfony.com/src/LiveMemory/Asset/CssCompiler.php [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+ */
+
+namespace App\LiveMemory\Asset;
+
+use Symfony\Component\AssetMapper\AssetMapperInterface;
+use Symfony\Component\AssetMapper\Compiler\AssetCompilerInterface;
+use Symfony\Component\AssetMapper\MappedAsset;
+use Symfony\Component\Filesystem\Path;
+
+class CssCompiler implements AssetCompilerInterface
```
Comment (weaverryan) [inline ux.symfony.com/src/LiveMemory/Asset/CssCompiler.php:27]: Could you just add a short description of the purpose of the compiler?

Comment (smnandre) [reply ux.symfony.com/src/LiveMemory/Asset/CssCompiler.php:27]: Done!

**Comment by weaverryan** [comment]
Thanks Simon!

**Pull request merged**